tests/Fixtures/TestBundle/Entity/Dummy.php

```diff
@@ -329,6 +336,22 @@ public function setDummyBoolean($dummyBoolean)
         $this->dummyBoolean = $dummyBoolean;
     }
 
+    /**
+     * @return bool
+     */
+    public function isDummyBoolean()
+    {
+        return $this->isDummyBoolean;
+    }
+
+    /**
+     * @param bool $isDummyBoolean
+     */
+    public function setIsDummyBoolean($isDummyBoolean)
+    {
+        $this->isDummyBoolean = $isDummyBoolean;
+    }
+
```

Comment:
Changing existing tests is not recommended, I hope that using `isDummyBoolean` on the property still works.

Reply:
The `isDummyBoolean` method was not taken for the schema generation, only the property.
I don't think the tests have changed.